src/pages/AddressInfoPage.tsx

```diff
@@ -127,54 +135,59 @@ const TransactionInfoPage = () => {
     fetchTransactions()
   }, [client, id, pageNumber])
 
+  const handleExportModalOpen = () => setExportModalShown(true)
+
+  const handleExportModalClose = () => setExportModalShown(false)
+
   if (!id) return null
 
   return (
     <Section>
-      <SectionTitle title="Address" />
-
-      <Table bodyOnly minHeight={250}>
-        <TableBody tdStyles={AddressTableBodyCustomStyles}>
-          <TableRow>
-            <td>Address</td>
-            <HighlightedCell textToCopy={id} qrCodeContent={id}>
-              {id}
-            </HighlightedCell>
-          </TableRow>
-          <TableRow>
-            <td>Number of Transactions</td>
-            <td>
-              {txNumberLoading ? (
-                <LoadingSpinner size={14} />
-              ) : (
-                txNumber ?? <ErrorMessage>Could not get total number of transactions</ErrorMessage>
-              )}
-            </td>
-          </TableRow>
-          <TableRow>
-            <td>Total Balance</td>
-            <td>
-              {totalBalanceLoading ? (
-                <LoadingSpinner size={14} />
-              ) : totalBalance ? (
-                <Badge type={'neutralHighlight'} amount={totalBalance.balance} />
-              ) : (
-                <ErrorMessage>Could not get balance</ErrorMessage>
-              )}
-            </td>
-          </TableRow>
-          {totalBalance?.lockedBalance && parseInt(totalBalance.lockedBalance) > 0 && (
+      <SectionTitle title="Address" subtitle={<HighlightedHash text={id} textToCopy={id} />} />
+      <TableAndQRCode>
+        <Table bodyOnly minHeight={100}>
+          <TableBody tdStyles={AddressTableBodyCustomStyles}>
             <TableRow>
-              <td>Locked Balance</td>
+              <td>Number of Transactions</td>
               <td>
-                <Badge type={'neutral'} amount={totalBalance.lockedBalance} />
+                {txNumberLoading ? (
+                  <LoadingSpinner size={14} />
+                ) : (
+                  txNumber ?? <ErrorMessage>Could not get total number of transactions</ErrorMessage>
+                )}
               </td>
             </TableRow>
-          )}
-        </TableBody>
-      </Table>
+            <TableRow>
+              <td>Total Balance</td>
+              <td>
+                {totalBalanceLoading ? (
+                  <LoadingSpinner size={14} />
+                ) : totalBalance ? (
+                  <Badge type={'neutralHighlight'} amount={totalBalance.balance} />
```

Comment:
`type="neutralHighlight"`

Reply:
Done! Added a couple of corresponding eslint rules.